.circleci/config.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       # Push the Docker image to DockerHub
       - run:
           name: Push Docker image to DockerHub
-          command: docker push openmf/ph-ee-bulk-processor:latest        
+          command: docker push openmf/ph-ee-bulk-processor:latest
```

Comment:
Though these changes are not harmful, best practice is to see `gitdiff` and avoid changes that are not yours.
these can sometimes create unnecessary merge conflicts and flip-flop formatting between yours and others changes.